src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -149,8 +149,10 @@ public static ShooterSpeeds calculateShooterSpeeds(double distance, boolean uppe
     // periodically update the values of motors for shooter to SmartDashboard
     @Override
     public void periodic() {
-        SmartDashboard.putNumber("shooter/left_rpm", getLeftRpm());
-        SmartDashboard.putNumber("shooter/right_rpm", getRightRpm());
+        SmartDashboard.putNumber("shooter/leftRpm", getLeftRpm());
+        SmartDashboard.putNumber("shooter/rightRpm", getRightRpm());
+        SmartDashboard.putNumber("shooter/leftCurrent", m_leftShooterMotor.getOutputCurrent());
+        SmartDashboard.putNumber("shooter/rightCurrent", m_rightShooterMotor.getOutputCurrent());
```

Comment:
Do we want to also log the feeder speed and current to see if the current will tell us if we have the note?